# Expected behavior
# Set (all=True and from_provided_tag=False) if entire history should be returned
# Set (all=True and from_provided_tag=True) for entire history from tag to current
#       tag must be valid!
# Set (all=False and from_provided_tag=True) for the tag entry specified
# Set (all=False and from_provided_tag=False) for returning the current entry
bool all
bool from_provided_tag
evd_ros_core/Version tag
---
string data
evd_ros_core/Version tag
bool status
string errors
string message
